kb/temporal-cluster-install-options.md

```diff
@@ -0,0 +1,93 @@
+---
+slug: install-temporal-clusters-for-development
+title: Install Temporal Clusters for development
+tags:
+  - kb-article
+date: 2022-06-07T00:00:00Z
+---
+
+The following section list different methods of deploying your [Temporal Clusters](/concepts/what-is-a-temporal-cluster) locally, so that you can use and interact with the [Temporal Client](/concepts/what-is-a-temporal-client) APIs and [tctl](/tctl) commands to test and develop applications.
```

Comment:
```suggestion
The following sections list various methods of deploying your [Temporal Clusters](/concepts/what-is-a-temporal-cluster) locally, so that you can use and interact with the [Temporal Client](/concepts/what-is-a-temporal-client) APIs and [tctl](/tctl) commands to test and develop applications.
```